test/fixtures/discovery/host/expected_published_host_discovery.json

```diff
@@ -13,6 +13,7 @@
         "cpu_count": 2,
         "socket_count": 1,
         "total_memory_mb": 4096,
-        "agent_version": "trento-agent-version"
```

Comment:
I'm just curious, this fixture are some api response? Or we use them in order to deep match a struct, like serialize the struct and deep match this fixtures in test?

Reply:
This json is what the agent is supposed to send to the web side, to the `/api/collect` endpoint.

Reply:
Thanks for clarification! I'm still exploring the codebase 👀